packages/myst-cli/src/build/init.ts

```diff
@@ -13,6 +13,19 @@ import { githubCurvenoteAction, githubPagesAction } from './gh-actions/index.js'
 import { getGithubUrl } from './utils/github.js';
 import { checkFolderIsGit } from './utils/git.js';
 const VERSION_CONFIG = '# See docs at: https://mystmd.org/guide/frontmatter\nversion: 1\n';
+import { binaryName, homeURL, readableName, baseConfigs } from '../utils/whiteLabelling.js';
+
+const extendsConfigItems = baseConfigs()
+  .map(
+    (item) => `
+    - ${item}`,
+  )
+  .join('');
+const extendsConfig = extendsConfigItems
+  ? `
+# ${readableName()} base configuration
+extend:${extendsConfigItems}`
+  : '';
```

Comment:
I think we remove this for now. Happy for this to come back, but I think we do it later.

```suggestion
import { binaryName, homeURL, readableName } from '../utils/whiteLabelling.js';
```